### PR TITLE
[ip-address] remove ip-address; types were inlined

### PR DESCRIPTION
**Pull request by beaugunderson**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test YOUR_PACKAGE_NAME`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If removing a declaration:
- [x] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [x] Delete the package's directory.
- [x] Add it to `notNeededPackages.json`.

**I want to highlight two documentation issues here as well;** the `README` suggests running `npm run test` to verify the removal, but I get this output:

```
$ npm run test

> definitely-typed@0.0.3 test /Users/beau/p/DefinitelyTyped
> dtslint types

Error: ENOENT: no such file or directory, open '/Users/beau/p/DefinitelyTyped/types/index.d.ts'
```

~~I looked at the other available scripts, and figured `compile-scripts` may fix the above error, but that also fails (even when I've stashed my changes):~~

```
$ npm run compile-scripts

> definitely-typed@0.0.3 compile-scripts /Users/beau/p/DefinitelyTyped
> tsc -p scripts

error TS5055: Cannot write file '/Users/beau/p/DefinitelyTyped/scripts/not-needed.js' because it would overwrite input file.

error TS5055: Cannot write file '/Users/beau/p/DefinitelyTyped/scripts/update-codeowners.js' because it would overwrite input file.

Found 2 errors.
```

**Update:** I think the `README` should be updated to suggest `npm run test-all` when a package is removed, not `npm run test`. :)

### CODE REVIEW

**Review by icopp — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -2539,7 +2539,6 @@
 /types/iost-contract/                                   @monaka
 /types/iota.lib.js/                                     @fogsh
 /types/ip/                                              @codeanimal @BendingBender
-/types/ip-address/                                      @icopp @danwbyrne
```
Comment (peterblazejewicz) [inline .github/CODEOWNERS:2542]: IMO this is automatic, done after removal is done, please amned this to not introduce a change

Comment (beaugunderson) [reply .github/CODEOWNERS]: @peterblazejewicz fixed 👍

File: README.md [hunk 2]
```diff
@@ -282,7 +282,7 @@ You can remove it by running `npm run not-needed -- typingsPackageName asOfVersi
 - `libraryName`: Name of npm package that replaces the Definitely Typed types. Usually this is identical to "typingsPackageName", in which case you can omit it.
 
 Any other packages in Definitely Typed that referenced the deleted package should be updated to reference the bundled types.
-You can get this list by looking at the errors from `npm run test`.
+You can get this list by looking at the errors from `npm run test-all`.
```
Comment (amcasey) [inline README.md:285]: Seems right.

**Review by amcasey — APPROVED** [review]

**Review by peterblazejewicz — APPROVED** [review]

**Review by icopp — APPROVED** [review]

**Review by peterblazejewicz — APPROVED** [review]

**Review by amcasey — APPROVED** [review]

**Comment by beaugunderson** [comment]
Ready to merge

**Pull request merged**